executor_manager/executors/docker/executor.py

```
@@ -504,3 +572,53 @@ def _call_callback(
+            "valid": valid,
+            "errorMessage": error_message,
+        }
+
+        try:
+            with httpx.Client(timeout=10.0) as client:
+                response = client.post(update_url, json=update_payload)
+                if response.status_code == 200:
+                    logger.info(f"Reported validation stage: {validation_id} -> {stage} ({progress}%)")
+                else:
+                    logger.warning(f"Failed to report validation stage: {response.status_code} {response.text}")
+        except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)